docs/2.9.0/docs/canton/usermanual/performance.rst

```diff
@@ -506,74 +506,104 @@ If you submit a ledger API command (which can have multiple Daml commands), then
-         -- ship a single view with the top node of this choice if the observer of the 
-         -- choice aligns with the observer of the contract.       
-         observer obs
-         controller owner 
+         -- ship a single view with the top node of this choice if the observer of the
+         -- choice aligns with the observer of the contract. In other words, either they are the same or the new observer
+         -- is hosted by any of the contract's informee participants.
+         observer new_obs
+         controller owner
          do
          forA_ batch (`exercise` Foo)
 
-
-As a rule, the number of views should depend on the number of stakeholder groups you have in your batch choice, 
-not the number of "batches" you process in parallel within one command.
+As a rule, the number of views should depend on the number of participant stakeholder groups
+you have in your batch choice that is not a subset of the parent's contract informee participants,
+not the number of "batches" you process in parallel within one command. A participant stakeholder group is formed
+by aggregating the participants that host each stakeholder, or, in other words, the set participants that
+need to see a particular view.
 
 The informees for the different type of transaction tree nodes are (also see :ref:`da-model-projections`):
-   * create: signatories, observers 
+   * create: signatories, observers
    * consuming exercise: signatories, observers, stakeholders of the choice (controller, choice observers)
    * nonconsuming exercise: signatories, stakeholders of the choice (controller, choice observers), but not the observers of the contract
    * fetch: signatories + actors of the fetch, which are all stakholders which are in the authorization context that the fetch executed in
    * lookupByKey: only key maintainers
 
+
+Strategy for Protocol Version 5 (and lower)
++++++++++++++++++++++++++++++++++++++++++++
+
+For protocol versions 5 and lower, Canton creates a view for every action node in the transaction tree if the informees
+or confirming parties change between the node and its parent. Therefore, in order to minimize the number of views,
+you should try to minimize the number of changes of informees and confirming parties.
+
+When compared with the previous decomposition strategy, this view decomposition is more granular,
+with more views created, and thus less performant, since the condition to merge an action node in a view
+is more restricted. In this case, we only consider the list of informees and any change to this list will spawn
+a new view.
+
+To batch, the rule here is to group the operations on the ledger based solely on the list of informees.
+You can use choice observers to align informees if necessary. With the previous model as reference, the batch
+function would work if the observer of the aggregating choice is the same as the `Example` contract observer
+(`obs` == `new_obs`).
```

Comment:
Given that `new_obs` is unbound in the above example, it's not clear what you are aiming for. In particular, I don't see why the above example would not work with PV <= 5. Can you elaborate on the difference?

Reply:
The idea is if the `obs` in  `doUpdate efficient owner obs batch = do` is different from the one in the '_Example_' contract it won't work for PV <= 5. For PV6 it will work even if the different observer is different as long as it is hosted in an informee participant of the '_Example_' contract.

Reply:
I remove the reference to `new_obs` and try to explain this in the text.